Makefile

```diff
@@ -473,10 +468,10 @@ publish-bintray-debian: $(PUBLISH_BINTRAY_DEBIAN)
 		-f $(publishable) \
 		-v $(APPLICATION_VERSION_DEBIAN) \
 		-r $(TARGET_ARCH) \
-		-t $(RELEASE_TYPE) \
 		-o $(BINTRAY_ORGANIZATION) \
 		-p $(BINTRAY_REPOSITORY_DEBIAN) \
-		-c $(BINTRAY_COMPONENT)))
+		-c $(BINTRAY_COMPONENT) \
+		-d $(BINTRAY_PACKAGE_DISTRIBUTION)))
```

Comment:
I guess an alternative approach, which _might_ be more consistent with the way we handle `ARCH` and `PLATFORM`, would be to just pass the 'base' SNAPSHOT / PRERELEASE value into the S3-publish and Bintray-publish scripts, and then have them use a `releasetype-convert.sh` script, similar to the `architecture-convert.sh` and `platform-convert.sh` scripts? \*shrug\*
e.g. `scripts/build/relasetype-convert.sh -r stable -t S3`could return `resin-production-downloads` and `scripts/build/relasetype-convert.sh -r prerelease -t bintray` could return `devel` ?

Hmmm, although that's probably one level of abstraction too much, as there's actually **two** variables which you need to modify for Bintray when switching between prerelease and stable versions... :-/
...so I guess you could have:
```
scripts/build/relasetype-convert.sh -r stable -t S3_bucket -> resin-production-downloads
scripts/build/relasetype-convert.sh -r prerelease -t bintray_distribution -> devel
scripts/build/relasetype-convert.sh -r prerelease -t bintray_component -n etcher -> etcher-devel
```
Yeah, that's kinda ugly and not very extensible; ignore me and just leave it all in the `Makefile` ;-)